src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -45,13 +46,22 @@
  *     filter on contamination fractions. Alternatively, provide a numerical fraction to filter with --contamination_fraction_to_filter.
  * </p>
  *
- * <h3>Example</h3>
+ * <h3>Input</h3>
```

Comment:
--> <h3>Usage example</h3>

I think we can remove lines 50-59 altogether.

Reply:
My bad. The input and output sections appear to be part of what we specify. Please ignore comment.

Reply:
OK